src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -791,8 +784,8 @@ public unsafe void CopyTo(Array array, int index)
 
                 Span<byte> span = byteArray.AsSpan(index);
 
-                int quotient = Div4Rem(arrayLength, out int remainder);
-                for (int i = 0; i < quotient; i++)
+                int fullInts = Div4Rem(arrayLength, out int extraBytes);
```

Comment:
Renamed these variables so it's more readable

Reply:
Please retain the original terminology -- `quotient` and `remainder` is standard terminology when it comes to integer division. Also, it's good if the change focused on the fix at hand rather than introducing unrelated refactoring.